modules/ocf_mirrors/manifests/init.pp

```diff
@@ -45,6 +47,10 @@
       source => 'puppet:///modules/ocf_mirrors/README.html',
       owner  => mirrors,
       group  => mirrors;
+
+    '/usr/local/sbin/record-mirrors-stats':
+      content => template('ocf_mirrors/record-mirrors-stats.py.erb'),
+      mode    => '0644';
```

Comment:
if it's in sbin it should be executable. additionally, if it has secrets, it shouldn't be world-readable and should have `show_diff => false`

ideally don't template a python file though. store the config separately.